build.sbt

```diff
@@ -197,13 +197,14 @@ lazy val zincCompile = (project in file("zinc-compile"))
   )
   .configure(addSbtUtilTracking)
 
-// Persists the incremental data structures using SBinary
+// Persists the incremental data structures using Protobuf
 lazy val zincPersist = (project in internalPath / "zinc-persist")
   .dependsOn(zincCore, zincCore % "test->test")
   .configure(addBaseSettingsAndTestDeps)
   .settings(
     name := "zinc Persist",
-    libraryDependencies += sbinary
+    compileOrder := sbt.CompileOrder.Mixed,
```

Comment:
Java interfaces depend on Scala code to provide the default implementations of certain entities. This is extensively used in all the Zinc modules and it's a legit use case. This is the only way to provide a Java-friendly API.